interface/patient_file/summary/demographics.php

```diff
@@ -169,7 +169,10 @@ function image_widget($doc_id,$doc_catg)
 
 <head>
 <?php html_header_show();?>
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
```

Comment:
Also important to note that we previously decided to place the $css_header above bootstrap.